src/main/java/org/zalando/nakadi/service/CursorsService.java

```diff
@@ -91,27 +89,22 @@ public CursorsService(final SubscriptionDbRepository subscriptionRepository,
             throws ServiceTemporarilyUnavailableException, InvalidCursorException, InvalidStreamIdException,
             NoSuchEventTypeException, InternalNakadiException, NoSuchSubscriptionException, UnableProcessException,
             AccessDeniedException {
-        final Scope commitScope = TracingService.activateSpan(parentSpan, false);
+        TracingService.activateSpan(parentSpan, false)
+                .setTag("subscription", subscriptionId)
+                .setTag("stream.id", streamId);
         try {
             final Subscription subscription = subscriptionCache.getSubscription(subscriptionId);
-
             authorizationValidator.authorizeSubscriptionView(subscription);
             authorizationValidator.authorizeSubscriptionCommit(subscription);
             validateSubscriptionCommitCursors(subscription, cursors);
-            TracingService.setCustomTags(commitScope.span(), ImmutableMap.<String, Object>builder()
-                    .put("subscription", subscriptionId)
-                    .put("stream.id", streamId)
-                    .build());
             final ZkSubscriptionClient zkClient = zkSubscriptionFactory.createClient(
                     subscription, LogPathBuilder.build(subscriptionId, streamId, "offsets"));
-
             validateStreamId(cursors, streamId, zkClient, subscriptionId);
-
             return zkClient.commitOffsets(
                     cursors.stream().map(cursorConverter::convertToNoToken).collect(Collectors.toList()),
                     new SubscriptionCursorComparator(new NakadiCursorComparator(eventTypeCache)));
         } catch (Exception e) {
-            TracingService.logErrorInSpan(commitScope, e.getMessage());
+            TracingService.logErrorInSpan(parentSpan, e.getMessage());
```

Comment:
what is the point to log this exception here, it could not be useful to client. the better is to log info about invalid stream id for example, otherwise user will see some Zookeeper related exception which does not make sense